src/schema/basic_slots.yaml

```diff
@@ -81,9 +81,11 @@ slots:
       - prov:endedAtTime
 
   git_url:
+    description: The url the points to the exact github location of a workflow.
```

Comment:
Small typo in here - The url "that" points to the exact github location....

Reply:
Thank you! I fixed it!